### ISSUE TITLE
display players class on profile

### ISSUE THREAD

**Issue by rilygreat** [HabitRPG/habitica-android#1846]
> Device: Xiaomi Redmi 8A Dual
> Android Version: 29
> AppVersion: Version 4.0.1 (4460)
> User ID: 7747905d-9592-441c-8e10-df377041a795
> Level: 47
> Class: warrior
> Is in Inn: false
> Uses Costume: false
> Custom Day Start: 0
> Timezone Offset: -330
> Details:
> 
>  Greetings,
> I have been noticing something different lately on player's bio. I figured out that in the android version of the app, the player class symbol is not displayed near the player level. Hope the team will fix this issue.

**Comment by saraolson** [comment]
we could easily add class after the player's level on their profile. the icon would be nice, but simply having 'Level 18 Rogue' would be enough for now

**Comment by AdamWier** [comment]
Hello, I would like to work on this for Hacktoberfest. To be clear, by profile, you mean the screen that is displayed when you click on an avatar in your party? And the text 'Level XX <<Class>>" should be displayed where we currently see "Level XX" below the picture of their avatar?

**Comment by saraolson** [comment]
@AdamWier thats right! i'll mark this issue as in progress for you. just let us know if you have to drop it for any reason :) thanks!

**Comment by AdamWier** [comment]
I've posted a pull request with a bit of an explanation :)
